docs/testing.md

```diff
@@ -40,6 +40,10 @@ To run just a subset of them, specify a particular file or directory:
     py.test tests/unit
     py.test tests/unit/api.py
 
+To run just a pep8 test:
+
+    py.test --pep8 -m pep8 *
```

Comment:
Sorry -- this breaks for me (see error below). I'd suggest either `py.test --pep8 -m pep8 .`,  `py.test --pep8 -m pep8 *.py haas/ tests/`, or what you had before (`py.test --pep8 -m pep8`).

I think what you had before is the simplest. Maybe add a comment above that says something like "(assumes your virtual env is in .venv)".
```
$ py.test --pep8 -m pep8 *
[...]
ERROR: not found: AUTHORS
(no name 'AUTHORS' in any of [])
ERROR: not found: COPYING
(no name 'COPYING' in any of [])
ERROR: not found: haas.wsgi
(no name 'haas.wsgi' in any of [])
ERROR: not found: LICENSE
(no name 'LICENSE' in any of [])
ERROR: not found: MANIFEST.in
(no name 'MANIFEST.in' in any of [])
ERROR: not found: setup.cfg
(no name 'setup.cfg' in any of [])
```